submissions/vovan-zt/Frogger-Game/app.js

```diff
@@ -61,11 +59,12 @@ Player.prototype.handleInput = function (keyPress) {
             this.x = 202;
             this.y = 405;
         }, 600);
-        count++;
     };
 };
 
-const allEnemies = [73, 156, 239].map(location =>location = new Enemy(location+50, location, 200));
+const enemyLocation = [73, 156, 239];   
+
+const allEnemies = enemyLocation.map(location =>location = new Enemy(location+50, location, 200));
```

Comment:
Observations:
(1) `const allEnemies = [...].map(...)` was good.
(2) Defining `enemyLocation = [73, 156, 239]` doesn't really help me as a peer developer. If I want to add an enemy what number do I add? Is there any correlation between these numbers and playfield rows?
(3) What are 50 and 200?

Connect numbers with user experience on a higher level. Make calculations based on concepts, like "row", "height", "width", "offset". Make as few baseline constants as possible. Let constants **speak** for themselves.

Reply:
I'm glad it works :)
I'll try to be more professional.

Reply:
Let's train this skill here. You will find it helpful in MPG and Friends App